src/main/resources/Resources/Language/ko_KR.yml

```diff
@@ -0,0 +1,185 @@
+# 한글화 BY mcsim415 (현 LS서버 어드민)
```

Comment:
Is the content in brackets your server?
You can write BY mcsim415
But please don't add ads
Also, thanks for your translation

Reply:
Ok. Sorry I'll remove that.

Reply:
I edited that. 
It's other story, I'd like to ask because I couldn't load the skin files from my server. If you feel uncomfortable talking here, I'll open the issue.

Reply:
discord is more suitable for discussion
https://discord.gg/pJjQDQC
Or use GitHub issues
https://github.com/lt-name/MurderMystery_Nukkit/issues